Classes/Views/CardCollectionViewCell.swift

```diff
@@ -9,54 +9,55 @@
 import UIKit
 
 class CardCollectionViewCell: UICollectionViewCell, UIGestureRecognizerDelegate {
-
+    
```

Comment:
@Huddie can you update your Xcode so these don't keep popping up?

<img width="796" alt="screen shot 2018-09-30 at 12 02 45 am" src="https://user-images.githubusercontent.com/739696/46253224-3624c600-c444-11e8-9492-fc99b8111f4d.png">

Reply:
So that's how you get rid of that!
@rnystrom Thanks!